Source/ROUtils/Utils/DragCubeTool.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Profiling;
+
+namespace ROUtils
+{
+    public class DragCubeTool : MonoBehaviour
+    {
+        private static readonly Dictionary<string, DragCube> _cacheDict = new Dictionary<string, DragCube>();
+        private static bool _statsRoutineStarted = false;
+        private static uint _cubesRenderedThisFrame = 0;
+        private static uint _cacheHitsThisFrame = 0;
+        private static long _elapsedTicks = 0;
+
+        private string _shapeKey;
+
+        public static bool UseCache { get; set; } = true;
```

Comment:
I'd rather have the cache be opt-in on a per-call level

Reply:
It already is opt-in by using the overload without `shapeKey` or just having the value as null.